cpp/zeder_importer.cc

```diff
@@ -0,0 +1,630 @@
+        }
+
+        if (not entry.parent_ppn_.empty()) {
+            current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::PARENT_PPN),
+                                    entry.parent_ppn_, "", IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+        }
+
+        if (not entry.parent_issn_print_.empty()) {
+            current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::PARENT_ISSN_PRINT),
+                                    entry.parent_issn_print_, "", IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+        }
+
+        if (not entry.parent_issn_online_.empty()) {
+            current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::PARENT_ISSN_ONLINE),
+                                    entry.parent_issn_online_, "", IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+        }
+
+        switch (type) {
+            case Zotero::HarvesterType::RSS:
+                current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::FEED), entry.primary_url_, "",
+                                IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+                break;
+            case Zotero::HarvesterType::CRAWL: {
+                current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::BASE_URL), entry.primary_url_, "",
+                                IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+
+                // insert other required keys if not present
+                std::string temp_buffer;
+                if (not current_section->lookup(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::MAX_CRAWL_DEPTH), &temp_buffer))
+                    current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::MAX_CRAWL_DEPTH), "1", "");
+
+                if (not current_section->lookup(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::EXTRACTION_REGEX), &temp_buffer))
+                    current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::EXTRACTION_REGEX), "", "");
+
+                break;
+            }
+            case Zotero::HarvesterType::DIRECT:
+                current_section->insert(Zotero::HARVESTER_CONFIG_ENTRY_TO_STRING_MAP.at(Zotero::HarvesterConfigEntry::URL), entry.primary_url_, "",
+                                IniFile::Section::DupeInsertionBehaviour::OVERWRITE_EXISTING_VALUE);
+                break;
+        }
+    }
+}
+
+
+bool DiffZederEntries(const ZederConfigData &old_config, const ZederConfigData &new_config,
+                      std::map<ZederEntry::Id, ZederConfigDiff> * const diffs, bool skip_global_timestamp_check = false)
```

Comment:
Unless you change the value of "skip_global_timestamp_check" in this function, please make it const.